publicRegistrationProcess.php

```diff
@@ -135,8 +132,6 @@
     }
 
     // Check publicRegistrationMinimumAge
-    $publicRegistrationMinimumAge = $settingGateway->getSettingByScope('User Admin', 'publicRegistrationMinimumAge');
-
     if (!empty($publicRegistrationMinimumAge) > 0 and $publicRegistrationMinimumAge > (new DateTime('@'.Format::timestamp($dob)))->diff(new DateTime())->y) {
```

Comment:
This line, although not part of your changes, seems somewhat suspect 🤔 `!empty($publicRegistrationMinimumAge) > 0` is technically valid, but not how one would normally handle a boolean. The time conversion is interesting too ... perhaps a task for future refactoring.